src/mumble/MainWindow.h

```diff
@@ -279,7 +282,17 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		/// Updates the user's image directory to the given path (any included
 		/// filename is discarded).
 		void updateImagePath(QString filepath) const;
-
+	signals:
+		void corkStream(const bool corked); 
```

Comment:
```suggestion
```

@davidebeatrici, this declaration isn't used anywhere and has no corresponding definition. I would recommend removing it.

Reply:
Thank you, I missed that.